sorting/insertion-sort/java/InsertionSort.java

```diff
@@ -0,0 +1,86 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class InsertionSort {
```

Comment:
A good practice is to add a comment just before the definition of each function that summarizes what that function does.